### PR TITLE
Set up "400: Bad Request" error on invalid routes

### PR DESCRIPTION
**Pull request by chia-yh**
Also set up collab service behind another route

### CODE REVIEW

File: api-gateway/default.conf [hunk 1]
```diff
@@ -80,14 +80,22 @@ server {
         proxy_pass http://history-service:8005/history/;
     }
 
-    location / {
```
Comment (YZTangent) [inline api-gateway/default.conf:83]: Should we also create an exact match directive to `/` with a hello world

**Review by YZTangent — COMMENTED** [review]
Propose to merge this change with Continuous Deployment Commit as this will cause merge conflicts

File: api-gateway/default.conf (hunk 1 above)
Comment (chia-yh) [reply api-gateway/default.conf:83]: Why

**Comment by chia-yh** [comment]
> Propose to merge this change with Continuous Deployment Commit as this will cause merge conflicts

Main concern is that this also handles error handling in the master branch; are the merge conflicts extensive?

**Review by GenFusion122 — APPROVED** [review]
lgtm

**Pull request merged**